--- I18N_UnicodeNormalizer-1.0.0/usr/share/pear/I18N/UnicodeNormalizer.php~	2007-09-07 10:04:38.646018340 +0200
+++ I18N_UnicodeNormalizer-1.0.0/usr/share/pear/I18N/UnicodeNormalizer.php	2007-09-07 10:25:54.518964740 +0200
@@ -53,7 +53,7 @@
  * @link      http://pear.php.net/package/I18N_UnicodeNormalizer
  */
 
-require_once 'UnicodeNormalizer/String.php';
+require_once 'I18N/UnicodeNormalizer/String.php';
 
 /**
  * Unicode Normalizer
@@ -764,4 +764,4 @@
     }
 }
 
-?>
\ Brak znaku nowej linii na końcu pliku
+?>
